docs/sources/operators-guide/configuring/about-configurations.md

```diff
@@ -15,6 +15,36 @@ A given configuration loads at startup and cannot be modified at runtime. Howeve
 
 To see the current configuration state of any component, use the [`/config`]({{< relref "../reference-http-api/index.md#configuration" >}}) or [`/runtime_config`]({{< relref "../reference-http-api/index.md#runtime-configuration" >}}) HTTP API endpoint.
 
+## Common configurations
+
+Some configurations, like object storage backend, are repeated for multiple components.
+To avoid the repetition in the configuration file, the [`common`]({{< relref "../configuring/reference-configuration-parameters/index.md#common" >}}) configuration section or `-common.*` CLI flags can be used.
+Common configurations are applied first to all the specific configurations, allowing them to be overridden later by specific values.
```

Comment:
```suggestion
Common configurations are first applied to all the specific configurations, which allows  them to be overridden later by specific values.
```
The are not overwritten, but overridden, right? If so, leave as is.

Reply:
I think they are overridden (actually, _their values_ are overridden).

Reply:
So there is no writing going on. :)